test/test_fakeredis.py

```diff
@@ -2112,6 +2112,37 @@ def test_zadd_incr(r, ch):
     assert zadd(r, 'foo', {'three': 1.0}, incr=True, xx=True, ch=ch) == 4.0
 
 
+def test_zpopmin_empty(r):
```

Comment:
It looks like all the unit tests will need a `@redis3_only` decorator since redis-py 2.x didn't implement the command.